babel_runner.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/venv python3
+"""Script for handling translations with Babel"""
+
+import argparse
+import os
+import subprocess
+import tomllib
+
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+# Global variables used by pybabel below
+DOMAIN = "messages"
+COPYRIGHT_HOLDER = "Python Software Foundation"
+LOCALES_DIR = os.path.relpath(os.path.join(PROJECT_DIR, "locales"))
+POT_FILE = os.path.relpath(os.path.join(LOCALES_DIR, f"{DOMAIN}.pot"), PROJECT_DIR)
+SOURCE_DIR = os.path.relpath(
+    os.path.join(PROJECT_DIR, "python_docs_theme"), PROJECT_DIR
+)
+MAPPING_FILE = os.path.relpath(os.path.join(PROJECT_DIR, "babel.cfg"), PROJECT_DIR)
```

Comment:
Perhaps prefix the config file with a dot?
```suggestion
MAPPING_FILE = os.path.relpath(os.path.join(PROJECT_DIR, ".babel.cfg"), PROJECT_DIR)
```

Reply:
Working on a GitHub Actions workflow setup to test the translation files. I renamed the mapping file to `.babel.cfg`. By the way, I was about to go crazy with babel\<tab> completing with both mapping file and script instead of only the script, so thanks for the suggestion!